packages/fuel-indexer/Cargo.toml

```diff
@@ -17,55 +17,31 @@ path = "src/bin/main.rs"
 anyhow = { version = "1.0", default-features = false }
 async-std = "1"
 async-trait = "0.1"
-axum = { version = "0.5", features = ["multipart"] }
 bincode = "1.3.3"
-cfg-if = "1.0"
 chrono = { version = "0.4", features = ["serde"] }
 clap = { version = "3.1", features = ["cargo", "derive", "env"] }
 fuel-core = { version = "0.17", optional = true }
 fuel-core-client = "0.17"
-fuel-indexer-api-server = { workspace = true, optional = true }
-fuel-indexer-database = { workspace = true }
-fuel-indexer-database-types = { workspace = true }
-fuel-indexer-lib = { workspace = true }
-fuel-indexer-metrics = { workspace = true, optional = true }
-fuel-indexer-postgres = { workspace = true }
-fuel-indexer-schema = { workspace = true, features = ["db-models"] }
-fuel-indexer-types = { workspace = true }
-fuel-tx = "0.26.0"
-fuel-types = "0.26.0"
+fuel-indexer-api-server = { version = "0.4", path = "../fuel-indexer-api-server", optional = true }
+fuel-indexer-database = { version = "0.4", path = "../fuel-indexer-database" }
+fuel-indexer-lib = { version = "0.4", path = "../fuel-indexer-lib" }
+fuel-indexer-schema = { version = "0.4", path = "../fuel-indexer-schema", features = ["db-models"] }
+fuel-indexer-types = { version = "0.4", path = "../fuel-indexer-types" }
```

Comment:
What's the reasoning for changing this from `{ workspace = true }`?

Reply:
@deekerno Nope. Bad rebase, will fix.